### PR TITLE
Fix part of #10: Hifi ContinuePlaying

### PR DESCRIPTION
**Pull request by Luffy18346**
## Explanation
Fix part of #10: Removed hardcoded height attribute and applied dimensionRatio of 16:9 to the image keeping a fixed width. So, the height of the image will be set according to the applied dimension ratio.

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [ ] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by rt4914** [comment]
@Luffy18346 please change the title to

`Fix part of #10: Hifi ContinuePlaying`

File: app/src/main/res/layout/ongoing_story_card.xml [hunk 1]
```diff
@@ -29,9 +29,10 @@
       <ImageView
         android:id="@+id/lesson_thumbnail"
         android:layout_width="80dp"
-        android:layout_height="44dp"
+        android:layout_height="0dp"
         android:layout_marginStart="8dp"
         android:importantForAccessibility="no"
```
Comment (rt4914) [inline app/src/main/res/layout/ongoing_story_card.xml]: Remove this and add content description to this image, which can be the story name

File: app/src/main/res/layout/ongoing_story_card.xml [hunk 2]
```diff
@@ -29,9 +29,10 @@
...
+        app:layout_constraintDimensionRatio="16:9"
         android:paddingStart="40dp"
```
Comment (rt4914) [inline app/src/main/res/layout/ongoing_story_card.xml]: Remove these paddings and update scale type to `android:scaleType="centerInside"`

**Review by rt4914 — CHANGES_REQUESTED** [review]
1. The divider color should be updated to: `#9C000000` the reason behind this is as follows:
The eight character color code in mocks contains first 6 characters as color and last two characters as percentage of transparency. Now the transparency in android takes first two character and also convert 61 to hex is 9C.

2. Bottom padding of reyclerview needs to be 172dp as per mocks.

File: app/src/main/res/layout/ongoing_story_card.xml (hunk 2 above)
Comment (Luffy18346) [reply app/src/main/res/layout/ongoing_story_card.xml]: I have removed extra paddings from right and left. and changed the `scaleType` to `centerInside`.

Comment (rt4914) [reply app/src/main/res/layout/ongoing_story_card.xml]: @Luffy18346 remove padding top too.

File: app/src/main/res/layout/ongoing_story_card.xml (hunk 1 above)
Comment (Luffy18346) [reply app/src/main/res/layout/ongoing_story_card.xml]: I have removed the attribute `importantForAccessibility` and added content description as per instruction. I also performed accessibility test to check whether the content description is heard or not, the test was successful and the content description is heard clearly when the user click on the list item.

**Comment by Luffy18346** [comment]
@rt4914 , I have looked at the mock and updated the bottom padding of reyclerView to 172dp.

**Comment by rt4914** [comment]
> @rt4914 , I have looked at the mock and updated the bottom padding of reyclerView to 172dp.

@Luffy18346 everything looks great, you just need to address two more comments:

1. https://github.com/oppia/oppia-android/pull/605#discussion_r367857241
2. https://github.com/oppia/oppia-android/pull/605#pullrequestreview-344402313

**Comment by Luffy18346** [comment]
@rt4914, I have not changed the divider color to `#9c000000` because that color is much darker than the divider color given in mock, I have checked by applying it in the app. Also, if we change this color value in color.xml then it will affect to every place where the color is used.

File: app/src/main/res/layout/ongoing_story_card.xml (hunk 2 above)
Comment (Luffy18346) [reply app/src/main/res/layout/ongoing_story_card.xml]: @rt4914, but after removing padding top, the image will not look same as in the mock I have checked.

Comment (rt4914) [reply app/src/main/res/layout/ongoing_story_card.xml]: That is a really good point but actually currently app contains dummy images and in real images will contain padding on its own therefore we need to remove this padding.

**Comment by rt4914** [comment]
> @rt4914, I have not changed the divider color to `#9c000000` because that color is much darker than the divider color given in mock, I have checked by applying it in the app. Also, if we change this color value in color.xml then it will affect to every place where the color is used.

Okay, even I was unsure about this. Sounds good. Thanks.

File: app/src/main/res/layout/ongoing_story_card.xml (hunk 2 above)
Comment (Luffy18346) [reply app/src/main/res/layout/ongoing_story_card.xml]: Removed padding top and finally pushed the code.

Comment (rt4914) [reply app/src/main/res/layout/ongoing_story_card.xml]: Looks good.

**Review by rt4914 — APPROVED** [review]
LGTM, thanks @Luffy18346

**Pull request merged**